tcf_website/static/common/send_feedback.js

```diff
@@ -0,0 +1,28 @@
+function postToDiscord(type, message) {
+    var data = {
+        type: type,
+        content: message
+    };
+
+    $.ajax({
+        type: "GET",
+        url: "/feedback/discord",
+        data: data
+    });
+}
+
+function sendEmail(subject, message, recipients) {
+    var data = {
+        subject: subject,
+        content: message,
+        recipients: recipients
```

Comment:
Are we expecting multiple email address here? Seems like it should be singular.

Reply:
It is possible to have multiple recipients, but right now it should only be one.